spec/services/csv_exporter_spec.rb

```diff
@@ -1,30 +1,53 @@
 require 'rails_helper'
 
 describe CSVExporter do
-  let(:late_team) { create :responding_team, name: 'Transport for London' }
-  let(:late_foi_case) { create :closed_case, :fully_refused_exempt_s40,
-                          :late,
-                          :flagged,
-                          name: 'FOI Case name',
-                          email: 'dave@moj.com',
-                          message: 'foi message',
-                          postal_address: nil,
-                          late_team_id: late_team.id
+  let(:late_team) {
+    create :responding_team,
+      name: 'Transport for London'
+  }
+
+  let(:late_foi_case) {
+    create :closed_case,
+      :fully_refused_exempt_s40,
+      :late,
+      :flagged,
+      name: 'FOI Case name',
+      email: 'dave@moj.com',
+      message: 'foi message',
+      postal_address: nil,
+      late_team_id: late_team.id
+  }
+
+  let(:sar_case) {
+    create :closed_sar,
+      name: 'SAR case name',
+      postal_address: "2 High Street\nAnytown\nAY2 4FF",
+      subject: 'Full details required',
+      email: 'theresa@moj.com',
+      message: 'my SAR message',
+      subject_full_name: 'Theresa Cant'
+  }
+
+  let(:extended_case) {
+    create :closed_case,
+      :fully_refused_exempt_s40,
+      :extended_for_pit,
+      name: 'FOI Case name',
+      email: 'dave@moj.com',
+      message: 'foi message',
+      postal_address: nil
   }
-  let(:sar_case) { create :closed_sar,
-                          name: 'SAR case name',
-                          postal_address: "2 High Street\nAnytown\nAY2 4FF",
-                          subject: 'Full details required',
-                          email: 'theresa@moj.com',
-                          message: 'my SAR message',
-                          subject_full_name: 'Theresa Cant'
+
+  # These columns can have arbitrary values as they are
```

Comment:
rather than removing them, I think we should be setting these values explicitly when we care about them

Reply:
They aren't being removed from checking. They are checked separately without having to override the FactoryBot create codes.

Reply:
The columns are still checked without having to explicitly set teams for the sake of the team_name which varies by Id number only.